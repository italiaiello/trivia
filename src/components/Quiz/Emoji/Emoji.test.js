import React from 'react'
import { shallow } from 'enzyme'
import Emoji from './Emoji'

describe("Emoji.js", () => {
    it('checks if it Emoji.js renders properly', () => {
        expect(shallow(<Emoji/>)).toMatchSnapshot()
    })
})


