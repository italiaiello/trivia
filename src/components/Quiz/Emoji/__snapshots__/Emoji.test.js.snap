// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Emoji.js checks if it Emoji.js renders properly 1`] = `ShallowWrapper {}`;
